internal/pkg/fixtures/local/empty-branch/.keboola/manifest.json

```diff
@@ -25,7 +25,7 @@
   },
   "branches": [
     {
-      "id": %%LOCAL_STATE_MAIN_BRANCH_ID%%,
+      "id": 1,
```

Comment:
Branch id všude na `1`

Reply:
@webrouse Mno, začínám řešit remote test a push do projektu už failuje na tom že branch s id `1` neexistuje. 😕 Tak teď přemýšlím co s tím, zase zpátky na reálnou branch? 🙂 I teda s ohledem na https://keboola.atlassian.net/browse/KAC-177. Nebo jsi to myslel že lokální test bude s tou fixturou s fixní branch id a remote test udělá tu samou práci co lokální test (use template) ale s reálnou branchí?

![CleanShot 2022-08-17 at 11 58 20@2x](https://user-images.githubusercontent.com/215660/185091555-a9ce9bcf-a74a-4a8d-987b-0f03a07d8e06.jpg)

Reply:
@JakubMatejka myslim, ze je to jednoduche:
- pre lokalne testy chces `branchId=1` a v dalsom kroku "staticke" hodnoty aj pre dalsie IDs
- pre remote testy musis pouzit ID existujucej (main) branch.

Riesenie: vo `fixture` si das placeholder, a v local teste tam das `1` a v remote teste, tam das `testProject.DefaultBranch().BranchID`.

Reply:
jo ok, nedošlo mi že se ten update šablony pro remote test bude volat znovu. díky.